python/meterpreter/meterpreter.py

```diff
@@ -777,6 +777,7 @@ def __init__(self, std, is_alive, name=None):
         self.is_alive = is_alive
```

Comment:
I left this as-is. It looks like it shadows the default `threading.Thread#is_alive` implementation, but it doesn't appear to be called from anywhere